iOS_SDK/OneSignalSDK/Source/OneSignalNotificationSettingsIOS10.m

```diff
@@ -74,6 +74,12 @@ - (void)getNotificationPermissionState:(void (^)(OSPermissionState *subscription
             status.answeredPrompt = settings.authorizationStatus != UNAuthorizationStatusNotDetermined && settings.authorizationStatus != provisionalStatus;
             status.provisional = (settings.authorizationStatus == 3);
             status.accepted = settings.authorizationStatus == UNAuthorizationStatusAuthorized && !status.provisional;
+            if (@available(iOS 14.0, *)) {
+                status.ephemeral = (settings.authorizationStatus == UNAuthorizationStatusEphemeral);
```

Comment:
`UNAuthorizationStatusEphemeral` is only available in Xcode 12. Since we need to build with older versions of Xcode to support bitcode from them we have to use the raw value by defining it as our on const or hard coded value here.

Reply:
Good catch! fixed this